src/main/java/nl/tudelft/context/controller/GraphController.java

```diff
@@ -13,9 +13,7 @@
 import org.apache.commons.collections.CollectionUtils;
 
 import java.net.URL;
-import java.util.List;
-import java.util.ResourceBundle;
-import java.util.Set;
+import java.util.*;
```

Comment:
*Auto-generated comment*

    Using the '.*' form of import should be avoided - java.util.*.